std_msgs/Header header

byte INITIALISE = 0
byte READY = 1
byte REACH_GPS = 2
byte FIND_ARTAG = 3
byte REACH_ARTAG = 4
byte APPROACH = 5
byte PASS_GATE = 6
byte DEINITIALISE = 7
byte ERROR = 8


byte state
